src/hrflow_connectors/connectors/talentsoft/warehouse.py

```diff
@@ -120,21 +123,96 @@ class ReadJobsParameters(ParametersModel):
     )
 
 
+class WriteProfileParameters(ParametersModel):
+    """Parameters for write action"""
```

Comment:
Ici c'est comme l'histoire des commentaires. 

Le nom de la class c'est `WriteProfileParameters`. Ton commentaire c'est `Parameters for write action` C'est quoi le gain ? 

Le nom indique déjà à quoi sert la classe.